src/utils/local_storage/updateDelegations.ts

```diff
@@ -0,0 +1,37 @@
+import { Delegation } from "@/app/types/delegations";
+
+import { filterDelegationsLocalStorage } from "./filterDelegationsLocalStorage";
+
+export const updateDelegations = async (
```

Comment:
```suggestion
export const calculateDelegationsDiff = async (
```

This does not do any updating.

Reply:
Done